bedrock/mozorg/templates/mozorg/contribute/contribute-tasks.html

```diff
@@ -0,0 +1,182 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "mozorg/contribute/contribute-base.html" %}
+
+{% add_lang_files "mozorg/contribute/index" %}
+
+{% block body_id %}inquiry{% endblock %}
+
+{% block page_title %}{{_('Get Involved: Volunteer Opportunities at Mozilla')}}{% endblock %}
+
+{% block contrib_nav_cta %}{% endblock %}
+{% block contrib_country_form %}{% endblock %}
+
+{% block contrib_title %}
+  <div class="container">
+    <h1 class="page-title">{{ _('Get involved with Mozilla today!') }}</h1>
+    <p class="section-tagline">{{ _('There are a lots of ways to contribute to Mozilla. Pick one below that you think would be right for you.') }}</p>
+  </div>
+{% endblock %}
+
+{% block contrib_content %}
+  {% if variation == 'a1' %}
+    <section class="section inquiry">
+      <div class="container inquiry-choices">
+        <div class="option-group">
+          <h4>{{ _('Only have a little time?') }}</h4>
+          <ul class="option-list">
+            <li class="option">
+              <a data-task-id="1" class="ab-task category-nightly" href="https://nightly.mozilla.org/" target="_blank"></a>
+              <span class="label-text">{{ _('Help us test by downloading Firefox Nightly') }}</span>
+            </li>
+            <li class="option">
+                <a data-task-id="2" class="ab-task category-stumbling" href="https://oneanddone.mozilla.org/en-US/tasks/92/" target="_blank"></a>
+                <span class="label-text">{{ _('Download our Stumbler App') }}</span>
+            </li>
+            <li class="option">
+              <a data-task-id="3" class="ab-task category-donate" href="https://sendto.mozilla.org/page/contribute/givenow-seq#page-1" target="_blank"></a>
+              <span class="label-text">{{ _('Donate to Mozilla') }}</span>
+            </li>
+            <li class="option">
+              &nbsp;
```

Comment:
The page was obviously designed for four options per row, so I can see why you added these empty items to make it work with the layout. But I think we could instead update the CSS to handle different cases where there are two or three items per row, adjusting widths to keep them centered.